packages/web/src/components/ui/SearchedResource/SearchedResource.tsx

```diff
@@ -22,7 +29,9 @@ export default function SearchedResource({ image, name, totalPackages, lastScanD
           {name} <span className={styles.searchedResourceHighlight}>{totalPackages} packages</span>
         </h3>
         {!!lastScanDate && (
-          <div className={styles.searchedResourceSubtitle}>Last scanning {lastScanDate}</div>
+          <div className={styles.searchedResourceSubtitle}>
```

Comment:
This is probably a good time to add a `Scan in progress...` during loading state

Reply:
This is related to #206 , not this PR